projects/igniteui-angular/src/lib/grids/common/crud.service.ts

```diff
@@ -97,10 +97,30 @@ export class IgxCell {
         public rowIndex: number,
         public column,
         public value: any,
-        public editValue: any,
+        public _editValue: any,
```

Comment:
Assuming `IgxCell` is internal (fairly certain it is), I guess this can remain public too. Could've been done with an extra method to assign the value too.